CONTRIBUTING.md

```diff
@@ -47,8 +48,8 @@ Before you submit your pull request (PR) consider the following guidelines:
 - Push your branch to GitHub.
 - Create a new pull request from your branch against the master branch.
 - The PR must be reviewed by [@thomas.pink](https://github.com/thomaspink),
```

Comment:
Can we change it to 
The PR must be reviewed by the code owners - Code owners will be added automatically to the PR for you!